src/release/process.md

```diff
@@ -18,55 +18,32 @@ time, but make sure the stable promotion lands first.
 
 ### Beta to stable
 
-Temporarily turn off GitHub branch protection for the `stable` branch in
-rust-lang/rust repo. In your local Rust repo:
+[Obtain AWS CLI credentials][awscli] and run this command from the [simpleinfra] repository:
 
-```sh
-$ git fetch origin
-$ git push origin origin/beta:stable -f
-# make sure that the release notes file is as fresh as possible
-$ git checkout origin/master -- RELEASES.md
+```
+./release-scripts/promote-branches.py
 ```
 
-Re-enable branch protection for the `stable` branch. Send a PR to rust-lang/rust
-on the stable branch making the following changes:
+Once that's done, send a PR to the freshly created beta branch of rust-lang/rust
+which updates `src/ci/channel` to `beta`.
+
+Also send a PR to rust-lang/rust targeting the new stable branch making the
+following changes:
 
 - Update `src/ci/channel` to `stable`
+- Update release notes to the latest available copy
+  * e.g., `git checkout origin/master -- RELEASES.md`
 
-Once the PR is sent, r+ it and give it a high `p=1000`.
+Once the PRs are sent, r+ both and give them a high `p=1000` (for stable) and
+`p=10` for beta.
 
 After the PR is merged you'll need to start a **dev** release. [Obtain AWS CLI
 credentials][awscli] and run this command from the [simpleinfra] repository:
 
 ```
-./start-release.py dev stable
+./release-scripts/start-release.py dev stable --release-date YYYY-MM-DD
```

Comment:
Would it be useful to have this default to the current date if you don't specify it?

Reply:
It should be the Thursday after for most releases, which seems annoying to predict and easy to miss needing to change. I think we should require it for now.